terraform/modules/infra-security-groups/main.tf

```diff
@@ -53,7 +68,184 @@ data "terraform_remote_state" "infra_networking" {
+# - downloading packages from package repos
+# - calling AWS APIs such as SSM, S3 and EC2
+# - scraping alertmanager on port 9093
+# - sending alerts to alertmanager on port 9093
+# - scraping external targets that run on the PaaS
+# - scraping itself on port 9090
+# - scraping node exporters on port 9100
+resource "aws_security_group_rule" "egress_from_prometheus_ec2_to_all" {
+  security_group_id = "${aws_security_group.prometheus_ec2.id}"
+  type              = "egress"
+  to_port           = 0
+  from_port         = 0
+  protocol          = "-1"
+  cidr_blocks       = ["0.0.0.0/0"]
+}
+
+resource "aws_security_group" "alertmanager_alb" {
+  name        = "${var.stack_name}-alertmanager-alb"
+  vpc_id      = "${data.terraform_remote_state.infra_networking.vpc_id}"
+  description = "Controls ingress and egress for the alertmanager alb"
+
+  tags = "${merge(
+    local.default_tags,
+    map("Stackname", "${var.stack_name}"),
+  )}"
+}
+
+resource "aws_security_group_rule" "ingress_from_office_http_to_alertmanager_alb" {
+  security_group_id = "${aws_security_group.alertmanager_alb.id}"
+  type              = "ingress"
+  from_port         = 80
+  to_port           = 80
+  protocol          = "tcp"
+  cidr_blocks       = ["${var.allowed_cidrs}"]
+}
+
+resource "aws_security_group_rule" "ingress_from_office_https_to_alertmanager_alb" {
+  security_group_id = "${aws_security_group.alertmanager_alb.id}"
+  type              = "ingress"
+  from_port         = 443
+  to_port           = 443
+  protocol          = "tcp"
+  cidr_blocks       = ["${var.allowed_cidrs}"]
+}
+
+resource "aws_security_group_rule" "egress_from_alertmanager_alb_to_alertmanager_ec2" {
+  security_group_id        = "${aws_security_group.alertmanager_alb.id}"
+  type                     = "egress"
+  to_port                  = 9093
+  from_port                = 9093
+  protocol                 = "tcp"
+  source_security_group_id = "${aws_security_group.alertmanager_ec2.id}"
+}
+
+resource "aws_security_group" "alertmanager_ec2" {
+  name        = "${var.stack_name}-alertmanager-ec2"
+  vpc_id      = "${data.terraform_remote_state.infra_networking.vpc_id}"
+  description = "Controls ingress and egress for alertmanager ec2 instances"
+
+  tags = "${merge(
+    local.default_tags,
+    map("Stackname", "${var.stack_name}"),
+  )}"
+}
+
+resource "aws_security_group_rule" "ingress_from_alertmanager_alb_to_alertmanager_ec2" {
+  security_group_id        = "${aws_security_group.alertmanager_ec2.id}"
+  type                     = "ingress"
+  from_port                = 9093
+  to_port                  = 9093
+  protocol                 = "tcp"
+  source_security_group_id = "${aws_security_group.alertmanager_alb.id}"
+}
+
+resource "aws_security_group_rule" "ingress_from_prometheus_ec2_to_alertmanager_ec2" {
+  security_group_id        = "${aws_security_group.alertmanager_ec2.id}"
+  type                     = "ingress"
+  from_port                = 9090
```

Comment:
ditto, I think this from_port should be `9093` too

Reply:
Explained understood and corrected thanks :-)